src/modules/sensors/sensors.cpp

```diff
@@ -156,6 +170,26 @@ class Sensors : public ModuleBase<Sensors>, public ModuleParams, public px4::Sch
 		int32_t air_cmodel;
 		float air_tube_length;
 		float air_tube_diameter_mm;
+
+		int32_t CAL_AV_AOA_ID;
+		float CAL_AV_AOA_OFF;
+		float CAL_AV_AOA_MIN;
+		float CAL_AV_AOA_MAX;
+		float CAL_AV_AOA_REV;
+		float CAL_AV_AOA_P0;
+		float CAL_AV_AOA_P1;
+		float CAL_AV_AOA_P2;
+		float CAL_AV_AOA_P3;
+
+		int32_t CAL_AV_SLIP_ID;
+		float CAL_AV_SLIP_OFF;
+		float CAL_AV_SLIP_MIN;
+		float CAL_AV_SLIP_MAX;
+		float CAL_AV_SLIP_REV;
+		float CAL_AV_SLIP_P0;
+		float CAL_AV_SLIP_P1;
+		float CAL_AV_SLIP_P2;
+		float CAL_AV_SLIP_P3;
```

Comment:
Would this fit better in a struct?

Reply:
It is already part of the Parameters strucht, right? I don't think making a struct in a struct makes sense.